### DISCUSSION TITLE
Sorting multiple columns by default

### DISCUSSION THREAD

**Discussion by DaedalusWings** [filamentphp/filament#5317] [Help] [answered]
Is it possible to sort multiple columns like this:
- First sort **year** (desc)
- Then sort **month** (asc)
- Then sort **day** (desc)

Its a stupid example but I hope it explains what I want to achieve.
I just found these in the docs: 

- [defaultSort()](https://filamentphp.com/docs/2.x/admin/resources/listing-records#sorting-a-column-by-default)
- [Sorting](https://filamentphp.com/docs/2.x/tables/columns/getting-started#sorting)

**Comment by danharrin** [comment, marked as answer]
In v3 its possible with `defaultSort()`
